locale/en-US/eas.dtd

```diff
@@ -25,20 +25,20 @@
 <!ENTITY pref.downloadonly "Revert local changes (one-way sync)">
 <!ENTITY pref.generaloptions "General options">
 <!ENTITY pref.syncdefaultfolders "Subscribe and sync all default folders after the initial connection has been established.">
-<!ENTITY pref.provision "Enforce provisioning (Kerio needs this)">
+<!ENTITY pref.provision "Enforce provisioning (Kerio requirement)">
 
 <!ENTITY pref.autotime "Sync interval (minutes)">
 <!ENTITY pref.never "A setting of 0 disables periodic sync. Push sync is no yet supported.">
-<!ENTITY pref.ShowTrashedFolders "Show folders found in trash" >
+<!ENTITY pref.ShowTrashedFolders "Show folders found in trash">
 
 <!ENTITY pref.syncstate "Sync status">
 <!ENTITY pref.seperator.comma "Comma">
 <!ENTITY pref.seperator.linebreak "Line break">
-<!ENTITY pref.seperator.description "Seperator for multiline address field.">
-<!ENTITY pref.displayoverride "Overide Display Name with 'First Name' + 'Second Name'">
+<!ENTITY pref.seperator.description "Separator for multiline address field.">
+<!ENTITY pref.displayoverride "Overide Display Name with “First Name” and “Second Name”">
```

Comment:
I want to keep the +

Reply:
Isn't "concatenate" what you doing. Only numbers can be added (+), not strings. It would be explicitly written as:
Overide Display Name with the concatenation of “First Name” and “Second Name”
or implicitly written as:
Overide Display Name with “First Name” and “Second Name”
However, reverting change to your preferred addition operator.